src/hash_to_curve/map_g1.rs

```diff
@@ -581,8 +581,6 @@ fn map_to_curve_simple_swu(u: &Fp) -> G1Projective {
 
 /// Maps an iso-G1 point to a G1 point.
 fn iso_map(u: &G1Projective) -> G1Projective {
-    const COEFFS: [&[Fp]; 4] = [&ISO11_XNUM, &ISO11_XDEN, &ISO11_YNUM, &ISO11_YDEN];
```

Comment:
out of curiosity: do you know what problems this caused?

Reply:
Some limitation of how position independent code works on the Ledger device - having a constant that references another constant doesn't work correctly. It's likely a toolchain bug (IMO), I don't know the details, the issue was described [on Slack](https://dfinity.slack.com/archives/C01LB4RUMMY/p1711485440205469?thread_ts=1709807694.518129&cid=C01LB4RUMMY) and I blindly suggested this change and it worked :rocket: